apis/io-engine/protobuf/v1/stats.proto

```diff
@@ -32,20 +32,21 @@ message NexusIoStatsResponse {
 // Common IO stats struct for all resource types.
 message IoStats {
   string name = 1; // Name of the device/resource.
-  uint64 num_read_ops = 2; // Number of read operations on the device.
-  uint64 bytes_read = 3; // Total bytes read on the device.
-  uint64 num_write_ops = 4; // Number of write operations on the device.
-  uint64 bytes_written = 5; // Total bytes written on the device.
-  uint64 num_unmap_ops = 6; // Number of unmap operations on the device.
-  uint64 bytes_unmapped = 7; // Total bytes unmapped on the device.
-  uint64 read_latency_ticks = 8; // Total read latency on the device in ticks.
-  uint64 write_latency_ticks = 9; // Total write latency on the device in ticks.
-  uint64 unmap_latency_ticks = 10; // Total unmap latency on the device in ticks.
-  uint64 max_read_latency_ticks = 11; // Max read latency in ticks occurred over num_read_ops.
-  uint64 min_read_latency_ticks = 12; // Min read latency in ticks occurred over num_read_ops.
-  uint64 max_write_latency_ticks = 13; // Max write latency in ticks occurred over num_write_ops.
-  uint64 min_write_latency_ticks = 14; // Min write latency in ticks occurred over num_write_ops.
-  uint64 max_unmap_latency_ticks = 15; // Max unmap latency in ticks occurred over num_unmap_ops.
-  uint64 min_unmap_latency_ticks = 16; // Min unmap latency in ticks occurred over num_unmap_ops.
-  uint64 tick_rate = 17; // Tick rate specific to node hosting the device.
+  string uuid = 2; // UUID of the device/resource.
```

Comment:
This is a breaking change..
It should be 18 and everything below remain the same

Reply:
uuid should go to the bottom?

Reply:
Doesn't need to, just needs to have number 18

Reply:
@tiagolobocastro As discussed,Since existing stats feature is not release. It should be okay to renumber protobuf fields.